src/main/webapp/jsp/adminSearch.jsp

```diff
@@ -21,3 +22,13 @@
     
     <t:statusMessage/>
 </ta:adminPage>
+<!-- Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )-->
+<c:set var="splittedSearchKey" value="${fn:split(data.searchKey, ' ')}" />		<!-- splitting by space	 -->
+<c:set var="formattedSearchKey" value='${fn:join(splittedSearchKey, \'","\')}' />				<!-- Joining with comma within single quotes -->
+<c:set var='formattedSearchKey' value='["${formattedSearchKey}"]' />									<!-- Surrounding string with single quote + square bracket -->
+
+<script type="text/javascript">
+ $( document ).ready(function() {
+	 $("body").highlight(${formattedSearchKey});
```

Comment:
@bamboo3250  I moved this code to adminSearch.js . I formatted the string in js file. Also corrected the indentation.